Scripts/Core/VRView.cs

```diff
@@ -342,7 +348,44 @@ void OnGUI()
 
             DoDrawCamera(guiRect);
 
-            Event e = Event.current;
+            MouseScrollDelta = Vector2.zero;
+            var e = Event.current;
+            MouseDelta = e.delta;
+            switch (e.type)
+            {
+                case EventType.ScrollWheel:
+                    MouseScrollDelta = e.delta;
+                    break;
+                case EventType.MouseDown:
+                    switch (e.button)
+                    {
+                        case 0:
+                            LeftMouseButtonHeld = true;
+                            break;
+                        case 1:
+                            RightMouseButtonHeld = true;
+                            break;
+                        case 2:
+                            MiddleMouseButtonHeld = true;
+                            break;
+                    }
+                    break;
+                case EventType.MouseUp:
+                    switch (e.button)
+                    {
+                        case 0:
+                            LeftMouseButtonHeld = false;
+                            break;
+                        case 1:
+                            RightMouseButtonHeld = false;
+                            break;
+                        case 2:
+                            MiddleMouseButtonHeld = false;
+                            break;
+                    }
+                    break;
+            }
+
```

Comment:
@AndrewTHEManeri, @dunity I found that the input manager just wasn't giving us mouse input at edit time. This was the best alternative I could think of. We should talk Rene about whether this is the case w/ the new input system.